distribution/Directory.Build.props

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Solution wide properties -->
+  <PropertyGroup Label="Assembly Naming">
+    <Company>CompanyName</Company>
+    <Authors>CompanyName</Authors>
+    <Copyright>© $(Company). All rights reserved.</Copyright>
+    <NeutralLanguage>en</NeutralLanguage>
+    <DefaultLanguage>en-US</DefaultLanguage>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Compile settings">
+    <Nullable>enable</Nullable>
+    <LangVersion>9.0</LangVersion>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1573,1591,1712</NoWarn>
+
+    <!-- Used by code coverage -->
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Analyzer settings">
```

Comment:
Maybe we should add a comment, that this is only relevent for .NET 3.1.x

Reply:
are you referring to DebugSymbols?

Reply:
No - the <AnalysisMode>AllEnabledByDefault</AnalysisMode>

Your checkin changed the line apparently..

Reply:
This is also applicable for net5 [see here](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#analysismode).

Reply:
> This is also applicable for net5 [see here](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#analysismode).

Cool - think I misread the documentation first time around :)